### ISSUE TITLE
Missing FieldException - Field "UnityModManagerNet.UnityModManager.modsPath' not found.

### ISSUE THREAD

**Issue by Yan-Zero** [newman55/unity-mod-manager#40]
MissingFieldException - Field "UnityModManagerNet.UnityModManager.modsPath' not found.

I consider the cause of this problem is that "modsPath" is Property now,but it is Field before.

( Mod was made based on UMM Version 0.12

**Comment by RedContritio** [comment]
I have the same problem that modsPath leads to exception.
Hope this could get fixed as quickly as possible.

**Comment by newman55** [comment]
Which mod? Is there a link?

**Comment by RedContritio** [comment]
there is it's code. (just related)
`
if (GUILayout.Button("Open Log File"))
{
        Process.Start($"{UnityModManager.modsPath}\\UnityModManager.log");
}
`
I just got it with dnSpy.

**Comment by newman55** [comment]
This variable is in place. Error occurs with the latest version of UMM?

**Comment by RedContritio** [comment]
[BaseResourceMod.zip](https://github.com/newman55/unity-mod-manager/files/4397274/BaseResourceMod.zip)
the language of this mod is simplified chinese, you may be hard to understand some words in it.
you could search "modsPath" to get where it is.
And when I use this mod, the error occurs. Just MissingFieldException as above.

**Comment by newman55** [comment]
Yes, it may have been a field before, in which case I don't know how to fix it.
If the mod is abandoned, I can try to adapt it to the latest version. But I'm not sure there won't be other mistakes.
[BaseResourceMod](https://www.dropbox.com/s/uymgdisyxgiqkxk/BaseResourceMod.zip?dl=1)

**Comment by RedContritio** [comment]
Maybe the source code does help.
[BaseResouceMod Source](https://github.com/phorcys/Taiwu_mods/tree/master/BaseResourceMod)

**Comment by newman55** [comment]
You can also delete this code and save the module via dnspy.
```csharp			
bool flag2 = GUILayout.Button("打开日志文件", new GUILayoutOption[]
{
	GUILayout.ExpandWidth(false)
});
if (flag2)
{
	Process.Start(UnityModManager.<modsPath>k__BackingField + "\\UnityModManager.log");
}
```

**Comment by RedContritio** [comment]
ok, I will have a try.
thanks for your help.

**Comment by newman55** [comment]
Check my link too.

**Comment by newman55** [comment]
You know that you can also download UMM 0.12 version?

**Comment by RedContritio** [comment]
Have downloaded your file.
I can use umm0.12, however, a new mod platform need higher version.
and, unfortunately, I have no rights to modify the platform's source code.

**Comment by newman55** [comment]
Fixed in 0.21.6
Thanks.
